_config.yml

```diff
@@ -1,10 +1,14 @@
 title: JKAN
 description: A lightweight, backend-free open data portal, powered by Jekyll.
 baseurl: /jkan
+logo:
```

Comment:
Is this intended to be a full `http://`-prefixed URL or a relative path like `/img/logo.png`?